addons/mail/static/src/models/chat_window_header_view.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'ChatWindowHeaderView',
+    identifyingFields: ['chatWindowOwner'],
+    fields: {
+        chatWindowOwner: one('ChatWindow', {
+            inverse: 'chatWindowHeaderView',
+            readonly: true,
+        }),
```

Comment:
Has this is an identifying field, it should be required

Reply:
Changes done